src/components/RedLiner/RedLiner.tsx

```diff
@@ -1,20 +1,35 @@
 import _ from 'lodash';
 import React, { useEffect, useRef, useState, RefObject } from 'react';
-import { Manager, Reference, Popper } from 'react-popper';
+import { Manager, Reference } from 'react-popper';
 
 import Line from '../Line/Line';
 import InfoBox from '../InfoBox/InfoBox';
 
 import styles from './RedLiner.module.scss';
 
-type dimensionType = 'height' | 'padding' | 'specs' | 'width';
+type dimensionType = 'height' | 'info' | 'width';
 
-interface IRedLinerProps {
-  /** The color to use */
+interface Config {
```

Comment:
Would prefer to call this `IConfig`, but docz displayed it as `Iconfig` in the Props table which didn't look right